data/mods/Magiclysm/Spells/biomancer.json

```diff
@@ -32,5 +32,78 @@
     "spell_class": "BIOMANCER",
     "difficulty": 4,
     "max_level": 1
+  },
+  {
+    "id": "vicious_tentacle",
+    "type": "SPELL",
+    "name": "Vicious Tentacle",
+    "description": "This spell extrudes a long nasty whiplike tentacle of sharp bones and oozing acid from your body, it has a long reach attack and vicious damage.",
+    "valid_targets": [ "self" ],
+    "min_range": 0,
+    "max_range": 0,
+    "min_damage": 1,
+    "max_damage": 1,
+    "effect": "spawn_item",
+    "effect_str": "tentacle_whip",
+    "base_casting_time": 350,
+    "base_energy_cost": 375,
+    "min_duration": 3000,
+    "max_duration": 9000,
+    "duration_increment": 3000,
```

Comment:
should this be 300?

Reply:
Actually, it should be around 200/200 as we were doing things previously, 50 per level of difficulty. I started making the summon item spells a little slower though. I must have slipped up on adjusting this, let's say 250/250 as that whip is rather nasty.

Reply:
i meant duration increment

Reply:
but no, i agree with the high cost, since 30 cutting is pretty incredible. it might actually be a little strong.

Reply:
I corrected the duration, you're right, a typo on my part.

As far as the 30 cutting goes, the jian sword was right below the bullwhip when I was looking at it, and it was bash8/cut30 as just a normal sword. Several swords (longsword/estoc) right near there go up to 38 cut and they're not even considered two-handed. I was thinking of this as one of the main weapons that would be used, I'm considering adding an acid projectile to the whip as well in fact...